packages/opentelemetry-exporter-prometheus/test/prometheus.test.ts

```diff
@@ -302,7 +302,7 @@ describe('PrometheusExporter', () => {
               assert.deepStrictEqual(lines, [
                 '# HELP counter a test description',
                 '# TYPE counter counter',
-                `counter{counterKey1="labelValue1"} 10 ${mockedTimeMS}`,
+                // `counter{counterKey1="labelValue1"} 10 ${mockedTimeMS}`,
```

Comment:
I expected to see this data in the output. This is happening due to below code:
https://github.com/open-telemetry/opentelemetry-js/blob/e37f96ab37e779e5d13aa3299df41ed54d14c2f3/packages/opentelemetry-exporter-prometheus/src/prometheus.ts#L185-L189

We need to find some better way to handle `Counter` in `prom-client`. #1039 is making more sense to implement.

Reply:
Would it make sense to tackle #1039 in this PR?

Reply:
My earlier assumption was wrong. Actually, this issue is caused by https://github.com/open-telemetry/opentelemetry-js/pull/1126/files#diff-9605a20eb27f2591c64bf9df2c518325R67. 

We should serialize `labels` object into string before adding it to the map (`_batchMap`). Could you please make this change and update the test?

Reply:
Great catch, not totally sure whats going on, lots of moving parts, but glad it works!